### DISCUSSION TITLE
wifi adapter is not up

### DISCUSSION THREAD

**Discussion by Falconcrest** [nzymedefense/nzyme#516] [Questions] [answered]
Hello, 

nzyme works great with my rpi4 4GB on DietPi Buster.
There are 4 wifi adapters connected.
But there is one problem.
After booting, the wifi adapters are not active.
I always have to start it with 4 commands:

sudo ip link set STICK01 up
sudo ip link set STICK02 up
sudo ip link set STICK03 up
sudo ip link set STICK04 up

Everything works fine for the next restart.

DietPi is an optimized Debian OS, the trick described under common problems does not work for Debian.
(/etc/netplan/01-standard.yaml....)

Is there another way under Debian to start the wifi adapter automatically after booting?

Best regards, Carsten

**Comment by lennartkoopmann** [comment, marked as answer]
I think this is a operating system specific question. I suggest asking in DietPi forums.

**Reply by Falconcrest** [reply]
Thanks for your answer. After a tip from the DietPi forum, I solved this with a shell script and a systemd service.
